openedx/core/djangoapps/user_authn/cookies.py

```diff
@@ -13,12 +13,70 @@
 from django.dispatch import Signal
 from django.utils.http import cookie_date
 
+from edx_rest_framework_extensions.auth.jwt import cookies as jwt_cookies
 from openedx.core.djangoapps.user_api.accounts.utils import retrieve_last_sitewide_block_completed
 from student.models import CourseEnrollment
 
+from .waffle import JWT_COOKIES_FLAG
+
+
 CREATE_LOGON_COOKIE = Signal(providing_args=['user', 'response'])
 
 
+LOGGED_IN_COOKIE_NAMES = (
+    # Header and payload sections of a JSON Web Token containing user
+    # information and used as an access token.
+    jwt_cookies.jwt_cookie_header_payload_name(),
+
+    # Signature section of a JSON Web Token.
+    jwt_cookies.jwt_cookie_signature_name(),
+
+    # Refresh token, which can be used to get a new JSON Web Token.
+    jwt_cookies.jwt_refresh_cookie_name(),
+
+    # TODO (ARCH-245): Remove the following deprecated cookies.
+    # Set to 'true' if the user is logged in.
+    settings.EDXMKTG_LOGGED_IN_COOKIE_NAME,
+
+    # JSON-encoded dictionary with user information.
+    settings.EDXMKTG_USER_INFO_COOKIE_NAME,
+)
+
+
+def is_logged_in_cookie_set(request):
+    """Check whether the request has logged in cookies set. """
+    if JWT_COOKIES_FLAG.is_enabled():
+        expected_cookie_names = LOGGED_IN_COOKIE_NAMES
+    else:
+        expected_cookie_names = (
+            settings.EDXMKTG_LOGGED_IN_COOKIE_NAME,
+            settings.EDXMKTG_USER_INFO_COOKIE_NAME,
+        )
+
+    return all(
+        cookie_name in request.COOKIES
+        for cookie_name in expected_cookie_names
+    )
+
+
+def delete_logged_in_cookies(response):
+    """
+    Delete cookies indicating that the user is logged in.
+    Arguments:
+        response (HttpResponse): The response sent to the client.
+    Returns:
+        HttpResponse
+    """
+    for cookie_name in LOGGED_IN_COOKIE_NAMES:
```

Comment:
If you are going to use it above, do you need to check `JWT_COOKIES_FLAG` here?

Reply:
According to the [docs](https://docs.djangoproject.com/en/2.1/ref/request-response/#django.http.HttpResponse.delete_cookie), delete_cookie "Fails silently if the key doesn’t exist."  That should be fine, right?